README.md

```diff
@@ -222,6 +223,8 @@ The descriptions of these items:
  * Typically, there will be almost no `require` definitions, except `php` version itself
  * The `php-ext` is a new top-level element to provide additional metadata for building the extension, if required.
    * Proposed JSON schema for this is in [composer-json-php-ext-schema.json](./composer-json-php-ext-schema.json)
+   * It is assumed that packages all support Zend Thread Safe mode (ZTS). If a package does **not** support ZTS mode,
+     the key `"support-zts": false` should be set in `php-ext` section
```

Comment:
Maybe add "you can also explicitly set `"support-zts": true` ?

Reply:
Added